src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -81,6 +84,17 @@ function WorkspaceInitialPage(props) {
         Navigation.goBack(ROUTES.SETTINGS_WORKSPACES);
     }, [props.reports, policy]);
 
+    useEffect(() => {
+        const policyDraftId = lodashGet(props.policyDraft, 'id', null);
+        if (!policyDraftId) {
+            return;
+        }
+
+        App.savePolicyDraftByNewWorkspace(props.policyDraft.id, props.policyDraft.name, '', false, '', !isSmallScreenWidth);
+        // I only care when the component renders the first time
```

Comment:
```suggestion
        // Call this function when component is mounted
```

Reply:
That is kind of the definition of `useEffect` that isn't explaining why I don't wanna add dependencies to the array.

Reply:
I understand, but I think we prefer third person (_it_), or first, but plural (_we_).